.github/scripts/build.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -eux
+
+. .github/scripts/env.sh
+. .github/scripts/common.sh
+
+uname -a
+uname -p
+uname
+pwd
+env
+
+# ensure ghcup
+if ! command -v ghcup ; then
+	install_ghcup
```

Comment:
Do you have a reason not to want to use `haskell/actions/setup` to get GHC and ghcup?

Reply:
Yes, it's unreliable.